napari/_app_model/constants/_commands.py

```diff
@@ -76,7 +76,7 @@ class CommandId(str, Enum):
     LAYER_PROJECT_MEDIAN = 'napari:layer:project_median'
 
     @property
-    def title(self) -> str:
+    def title(self) -> str: # type: ignore[override]
```

Comment:
```suggestion
    def title(self) -> str:  # type: ignore[override]
```
Has 1 space, switch to conventional 2 spaces.
I see mypy indeed does not recognize that it means to override, which would lead an error. I had a look to see what would change if we didn't have this as property, it leads to...other type errors in `ToggleViewerAction`. Could be that we need to have a look, but not part of this PR.

Reply:
Strange that black do not catch this. Based on this https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#comments it should be fixed by black in pre-commit.